### ISSUE TITLE
Not immediately clear how to close settings

### ISSUE THREAD

**Issue by painquin** [WillWilli/PlateUpEnhancementMod#28]
Pressing F1 a second time does not close the settings panel, like F2 does for the spawn panel, and there's no indication that clicking elsewhere on the screen will close the dialog. This is not a great experience for a first-time user.

**Comment by WillWilli** [comment]
I agree but unfortunately, the configuration manager is an external tool. It does mention (here)[https://github.com/BepInEx/BepInEx.ConfigurationManager/blob/da2fc62b987565d0296f9cbbc17789ca944fe92b/ConfigurationManager/ConfigurationManager.cs] that pressing F1 should turn the menu On & Off, so maybe this is a bug on their side. It looks like it's because the configuration manager automatically puts your cursor in the textbox when you click it. Maybe open this issue on their github 🤷
